NEWS.md

```diff
@@ -1,6 +1,24 @@
-# cchsflow 0.2.2 (Latest build)
+# cchsflow 0.2.3 (Latest build) 
+2019-11-29
 
-## Features:
+## Features
+
+- Updated example in the introduction (README.MD) to reflect data that is now included in the cchsflow package.
+- Updates to vignettes to reflect latest version of package
+- Updates to variables.csv & variableDetails.csv
+
+### Variables
+
+-  <font color="green">**New**</font> [`DHHGAGE_C`](reference/DHHGAGE_cat_fun.html) - categorical age variable that groups various age categories across all CCHS cycles. Based on the continuous age variable (`DHHGAGE_cont`) that is also harmonious across all CCHS cycles.
```

Comment:
The link is off it links to (news/reference/dh................)

Reply:
@wyusuf068 can you fix that?